test/integration/protoutils.go

```diff
@@ -38,24 +54,37 @@ func MakeProtoTarget(target string, path string) string {
 }
 
 // MakeProtoPath returns a path: element for a given target and path
-func MakeProtoPath(target string, path string) string {
+func MakeProtoPath(target string, path string, stripNamespaces bool) string {
 	var protoBuilder strings.Builder
 
 	protoBuilder.WriteString("path: ")
-	gnmiPath := MakeProtoTarget(target, path)
+	gnmiPath := MakeProtoTarget(target, path, stripNamespaces)
 	protoBuilder.WriteString(gnmiPath)
 	return protoBuilder.String()
 }
 
+func makeProtoValue(value string) string {
+	var protoBuilder strings.Builder
+
+	if strings.Contains(value, ":") {
```

Comment:
I think if you made the string value "123456" as "string_val:123456" above, you would not need this if statement at all. What if the string value you set had a MAC address or IPv6 address in it - you wouldn't be handling it right

Reply:
I split out the type and the value into separate entities. See if you like it.